metricbeat/module/windows/perfmon/pdh_windows.go

```diff
@@ -82,11 +82,25 @@ func PdhCloseQuery(query PdhQueryHandle) error {
 	return nil
 }
 
+type Counter struct {
+	handle PdhCounterHandle
+	format PdhCounterFormat
+}
+
+type Counters map[string]*Counter
+
 type Query struct {
 	handle   PdhQueryHandle
-	counters map[string]PdhCounterHandle
+	counters Counters
 }
 
+type Format int
+
+const (
+	FLOAT Format = iota
```

Comment:
I would probably call these `FloatFormat` and `LongFormat` since the names are rather generic. Constants should use upper-camel-case (I usually ignore this rule for constants brought over from C code).